src/components/Squiggly.module.css

```diff
@@ -33,6 +33,8 @@
 	transform: scale(1 1.17);
 }
 
-.squiggly:is(.alwaysActive, .passive, :focus, :hover):not(.inactive)::after {
+.squiggly:is(.alwaysActive, .passive, :focus-visible, :hover):not(
+		.inactive
+	)::after {
```

Comment:
[Tangent] This is some unfortunate formatting. Nothing we can do here 😄 🤷 just Prettier making a _choice_.